bin/omarchy-install-password-manager

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if (($# == 0)); then
+  echo "Usage: omarchy-install-password-manager [1password|bitwarden|keepassxc]"
+  exit 1
+fi
+
+package="$1"
+
+# Install package
+omarchy-pkg-add $package
```

Comment:
you could install both bitwarden and bitwarden-cli as it was previously, if the user chooses bitwarden